src/transactions/BumpSequenceOpFrame.cpp

```diff
@@ -42,6 +43,8 @@ BumpSequenceOpFrame::doApply(AbstractLedgerTxn& ltx)
     LedgerTxn ltxInner(ltx);
     auto header = ltxInner.loadHeader();
     auto sourceAccountEntry = loadSourceAccount(ltxInner, header);
+    tryUpdateAccountOnLedgerSeqUpdate(header, sourceAccountEntry);
+
```

Comment:
There's a bug here - `ltxInner` is only committed if there's a bump, but the CAP specifies that `seqLedger ' and `seqTime` should always be updated.

Reply:
Why does the CAP want that behavior?

Reply:
@sisuresh The reasoning is correct. We don't actually need this property for Starlight specifically, but it was a property that @stanford-scs wanted, but we maybe need to dig into why and explain it in the design rationale. Aside, it does seem simpler in my opinion to assume that if a tx containing a bump seq succeeds, it always clears `seqLedger`/`seqTime` given that we define bump seq's success as independent from whether the bump actually has any affect or not on the seq.